app/proguard-rules.pro

```diff
@@ -61,14 +61,5 @@
 
 -keep class com.github.mikephil.charting.** { *; }
 
--dontwarn com.evernote.android.job.gcm.**
--dontwarn com.evernote.android.job.util.GcmAvailableHelper
-
--keep public class com.evernote.android.job.v21.PlatformJobService
--keep public class com.evernote.android.job.v14.PlatformAlarmService
--keep public class com.evernote.android.job.v14.PlatformAlarmReceiver
--keep public class com.evernote.android.job.JobBootReceiver
--keep public class com.evernote.android.job.JobRescheduleService
-
```

Comment:
Check if there's proguard requirement in WorkManager

Reply:
I couldn't find any progaurd requirement for WorkManager.